LSPS1/README.md

```diff
@@ -185,13 +185,13 @@ The client MUST check if [option_support_large_channel](https://bitcoinops.org/e
   "payment": {
     "bolt11": {
       "state": "EXPECT_PAYMENT",
-      "fee_total_sat": 8888,
-      "order_total_sat": 2008888,
+      "fee_total_sat": "8888",
+      "order_total_sat": "2008888",
       "expires_at": "2015-01-25T19:29:44.612Z",
       "invoice" : "lnbc252u1p3aht9ysp580g4633gd2x9lc5al0wd8wx0mpn9748jeyz46kqjrpxn52uhfpjqpp5qgf67tcqmuqehzgjm8mzya90h73deafvr4m5705l5u5l4r05l8cqdpud3h8ymm4w3jhytnpwpczqmt0de6xsmre2pkxzm3qydmkzdjrdev9s7zhgfaqxqyjw5qcqpjrzjqt6xptnd85lpqnu2lefq4cx070v5cdwzh2xlvmdgnu7gqp4zvkus5zapryqqx9qqqyqqqqqqqqqqqcsq9q9qyysgqen77vu8xqjelum24hgjpgfdgfgx4q0nehhalcmuggt32japhjuksq9jv6eksjfnppm4hrzsgyxt8y8xacxut9qv3fpyetz8t7tsymygq8yzn05"},
```

Comment:
nit: While we are at it maybe put `},` on a newline to be consistent with other code format styles.